samples/contrib/aws-samples/debugger_component_demo/README.md

```diff
@@ -0,0 +1,57 @@
+# Simple pipeline with train component and debugger
```

Comment:
Check our other samples, but I have a feeling we usually do Proper Case for headers?